src/styles/shared/components/Buttons/_index.scss

```diff
@@ -0,0 +1,168 @@
+@use "sass:map";
+@use "@hypothesis/frontend-shared/styles/mixins/focus";
+@use './_mixins';
+@use "../../variables" as var;
+
+// Design tokens from variables
+$-settings: (
```

Comment:
I didn't realize that SASS had built in support for [private members](https://sass-lang.com/documentation/at-rules/use#private-members) by prefixing them with a hyphen or underscore. We should make more use of that.

Reply:
Yes, and more use of module naming with same.